jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -11,7 +11,7 @@
 
 from jdaviz.utils import PRIHDR_KEY
 
-ASTROPY_LT_5_3 = not minversion(astropy, "5.3.dev")
+ASTROPY_LT_5_3 = not minversion(astropy, "5.3")
```

Comment:
(Not strictly related to this PR, but since 5.3 was released, I have removed the `.dev`)